src/core/load-fragment.c

```diff
@@ -2744,6 +2744,16 @@ int config_parse_environ(
                 return 0;
         }
 
+        bool is_system = ltype;
+
+        const Specifier table[] = {
+                COMMON_SYSTEM_SPECIFIERS,
+                COMMON_TMP_SPECIFIERS,
+                COMMON_CREDS_SPECIFIERS(is_system ? LOOKUP_SCOPE_SYSTEM : LOOKUP_SCOPE_USER),
```

Comment:
use `u->manager->unit_file_scope`, and don't bother with ltype

Reply:
@poettering This is also used when parsing system.conf or user.conf, and in that case, `u` is NULL.

Reply:
I got the ltype idea from `config_parse_emergency_action`, which also deals with null unit. user.conf conf is my use case for this, so I'd like that to work.

Reply:
ah, fuck. makes sense.